src/plots/linePlot.ts

```diff
@@ -451,7 +451,7 @@ export class Line<X> extends XYPlot<X, number> {
 
     const chartBounds = this.bounds();
     this.entities().forEach((entity) => {
-      if (!this._entityVisibleOnPlot(entity, chartBounds)) {
+      if (!Utils.Math.within(entity.position, chartBounds)) {
```

Comment:
oh i guess we can just replace `this._entityVisibleOnPlot` with the math util in `plot.ts` too